test/version.ts

```diff
@@ -0,0 +1,7 @@
+import { DbVersion } from '../src/utils/dbVersion';
+
+const version = DbVersion.fromString(`v${process.env.WEAVIATE_VERSION!}`);
+
+/** Run the suite / test only for Weaviate version above this. */
+export const requireAtLeast = (...semver: [...Parameters<DbVersion['isAtLeast']>]) =>
+  version.isAtLeast(...semver) ? describe : describe.skip;
```

Comment:
Very happy with this abstraction! Hopefully we get a chance to move the other instances of `maybe` to use this

Reply:
Thank you!
I've been updating all the test files I needed to modify as I went, we'll soon update them all.

I was thinking about returning an object `{ describe, test }` at first, but so far haven't had the need to do anything like this:

```ts
requireAtLeast(1, 28, 4).describe('desc', () => {
  requireAtLeast(1, 29, 2).test('', () => {})
  requireAtLeast(1, 29, 5).test('', () => {})
})
```

Grouping all related tests under one "require" block seems sufficient.